conans/test/conftest.py

```diff
@@ -109,7 +109,13 @@
         "system": {"path": {'Windows': 'C:/bazel/bin',
                             "Darwin": '/Users/jenkins/bin'}},
     },
-    'premake': {},
+    'premake': {
```

Comment:
This will not work in our CI once you start using ``premake`` with ``pytest.mark.tool_premake`` in any test, because our CI agents will not have premake installed. We should do:

- Put it in the comments sections of this file, for reference
- Move it to your own ``conftest_user.py`` so it works for you locally.
- Add ``pytest.mark.tool_premake`` to the test
- Keep the ``disabled`` in the conftest.py until we add it to CI
- We will ask @davidsanfal if they can put premake5 in our CI builders, and report on the location